### ISSUE TITLE
JSON RPC support

### ISSUE THREAD

**Issue by slavonnet** [ktorio/ktor#1954]
Any chance to finaly add JSON RPC support  ?

i see pool request

https://github.com/ktorio/ktor/pull/587

**Comment by Ololoshechkin** [comment]
@cy6erGn0m , that pull-request has become 2 years ago because there were no activity on fixing this. Maybe we can continue the work by ourselves to add this feature?

**Comment by cy6erGn0m** [comment]
The main question is why do we need this and what is advantage of that over regular ktor routes + json feature?

**Comment by lamba92** [comment]
Also I'd point out, why not publish a regular standalone feature for that!

**Comment by slavonnet** [comment]
Many commercial enterprise solutions still know how to send SOAP or JSON RPC. I have made a crutch on retrofit and now my JSON RPC is working. Just the day before yesterday I received another microservice with JSON RPC hanging generally on a regular socket.

I came to this project because I want purely in Kotlin so that all the libraries are.

> Also I'd point out, why not publish a regular standalone feature for that!

I would like to use Ktor as a client and a mock server with emulation of answers for retrofit.
